cluster/gce/util.sh

```diff
@@ -2915,7 +2918,7 @@ function create-master() {
     --network "${NETWORK}" \
     --allow tcp:443 &
 
-  if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]] || [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
+  if [[ "${ENABLE_PROMETHEUS_DEBUG:-false}" == "true" ]]; then
     gcloud compute firewall-rules create "promethues-${MASTER_NAME}" \
```

Comment:
@vinaykul, I changed this section, 9090 is only for debug prometheus, it shouldn't be related scale-out, Please confirm.

Reply:
Looks fine to me, but @pdgetrf should confirm.